src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/MessageList.vue

```diff
@@ -1,3 +1,4 @@
+<!-- eslint-disable vue/no-v-html -->
```

Comment:
are we certain we would like to disable this?

Reply:
there is a problem in dynamics side. dynamics sends messages as HTML to backend, we have to use v-html to render them in FE and we get a warning in the frontend because of using this tag.